snekbox/api/resources/eval.py

```diff
@@ -36,8 +36,8 @@ class EvalResource:
                     "properties": {
                         "path": {
                             "type": "string",
-                            # Disallow single absolute paths, and null bytes
-                            "pattern": r"^[^/\\0].*",
+                            # Disallow starting with / or containing \0 anywhere
+                            "pattern": r"^(?!/)(?!.*\\0).*$",
```

Comment:
Absolute case seems to be covered in tests but we do need to test the null byte as well.

Reply:
Added additional tests in [55a9223](https://github.com/python-discord/snekbox/pull/159/commits/55a92231d83439b31441945799f713c25b569278) and checks to make sure separate `\` and `0` are allowed.